UITests/SearchTests.swift

```diff
@@ -164,11 +158,23 @@ class SearchTests: KIFTestCase {
     }
 
     override func tearDown() {
+        resetSuggestionsPrompt()
         do {
             try tester().tryFindingTappableViewWithAccessibilityLabel("Cancel")
             tester().tapViewWithAccessibilityLabel("Cancel")
         } catch _ {
         }
         BrowserUtils.clearHistoryItems(tester(), numberOfTests: 5)
+        BrowserUtils.resetToAboutHome(tester())
+    }
+}
+
+private extension KIFUITestActor {
```

Comment:
I did this as a local extension for now. I guess ideally this should be either put into a global extension or even contributed upstream.